routes/web.php

```diff
@@ -56,7 +56,7 @@
     });
 
     Route::group(['prefix' => 'nfts'], function () {
-        Route::post('{nft:token_number}/reports', [NftReportController::class, 'store'])->name('nft-reports.create')->middleware('throttle:nft:reports');
+        Route::post('{nft:id}/reports', [NftReportController::class, 'store'])->name('nft-reports.create')->middleware('throttle:nft:reports');
```

Comment:
Just FYI, you can just omit the parameter here, `{nft}/reports`, and it will default to the route key name (https://laravel.com/docs/10.x/routing#customizing-the-default-key-name), which is ID, but this is just fine to be explicit.